contracts/BaseStrategy.sol

```diff
@@ -239,15 +249,23 @@ abstract contract BaseStrategy {
      * would be negatively affected if `tend()` is not called shortly, then this can return `true`
      * even if the keeper might be "at a loss" (keepers are always reimbursed by yEarn)
      *
+     * NOTE: `callCost` must be priced in terms of `want`
```

Comment:
Should a strategy implement a function that quotes its `want` or should a keeper bot do this?

Reply:
We could add, but then we'd be forcing it to use one typical pricing function. Better to let the external user define that.